.github/workflows/ci-editor.yml

```
@@ -0,0 +1,149 @@
+        required: true
+        default: 'false'
+
+jobs:
+  ci-editor-core-test:
+    if: ${{ inputs.core == 'true' }}
+    defaults:
+      run:
+        working-directory: 'editor/core'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)